themes/src/main/resources/theme/base/admin/resources/partials/realm-events-config.html

```diff
@@ -91,7 +91,18 @@ <h2>{{:: 'events-config' | translate}}</h2>
                         <button class="btn btn-danger" type="submit" data-ng-click="clearAdminEvents()" >{{:: 'clear-admin-events' | translate}}</button>
                     </div>
                 </div>
-                
+                <div class="form-group" data-ng-show="eventsConfig.adminEventsEnabled">
+                    <label class="col-md-2 control-label" for="expiration">{{:: 'expiration' | translate}}</label>
+                    <kc-tooltip>{{:: 'events.expiration.tooltip' | translate}}</kc-tooltip>
+                    <div class="col-md-6 time-selector">
+                        <input class="form-control" type="number" data-ng-model="realm.attributes.adminEventsExpiration" id="adminEventsExpiration" name="adminEventsExpiration" min="0"/>
+                        <select class="form-control" name="adminEventsExpirationUnit" data-ng-model="realm.attributes.adminEventsExpirationUnit">
```

Comment:
I didn't notice this before but why is this an attribute of the realm instead of one that's on the event config?

Reply:
@edewit What kind of event config do you mean? The expiration config for user events is already a column on the REALM table. The configuration for admin events was implemented similarly, just using a realm attribute in order to not have a schema change.